product_profile/README.rst

```diff
@@ -0,0 +1,128 @@
+
+Additional feature: a default value can be attached to a profile (see § Configuration, part 3)
+
+
+Configuration
+=============
+
+1. Create your own profile here: Sales > Configuration > Product Categories and Attributes > Product Profiles
+
+.. image:: static/description/list.png
+
+
+2. To have more fields available to attach to this profile you must define
+   these fields in the model 'product.profile' in your own module
+   If the field name (and its type) is the same than those in 'product.template'
+   then values of these will be populated automatically
+   in 'product.template'
+   Example of fields declaration in your own module:
+   ```python
+class ProductProfile(models.Model):
+    """ Require dependency on sale, purchase and point_of_sale modules
+    """
+
+    _inherit = 'product.profile'
+
+    def _get_types(self):
+        return [('product', 'Stockable Product'),
+                ('consu', 'Consumable'),
+                ('service', 'Service')]
+
+    sale_ok = fields.Boolean(
+        string='Can be Sold',
+        help="Specify if the product can be selected in a sales order line.")
+    purchase_ok = fields.Boolean(
+        string='Can be Purchased')
+    available_in_pos = fields.Boolean()
+   ```
+
+3. Second behavior: you might want to add a default behavior to these fields:
+   in this case use prefix 'profile_default\_' for your field name
+   in 'product.profile' model.
+   ```python
+class ProductProfile(models.Model):
+    ...
+    profile_default_categ_id = fields.Many2one(
+        'product.category',
+        string='Default category')
+   ```
+   In this case 'categ_id' field (from product.template) is populated
+   with 'profile_default_categ_id' value but can be updated manually by the user.
+
+
+4. Insert data (xml or csv) and define values for each field defined above
+   for each configuration scenario
+
+
+Usage
+=====
+
+Assign a value to the profile field in the product template form.
+Then, all fields which depend on this profile will be set to the right value at once.
+
+If you deselect the profile value, all these fields will be reset to empty values.
+
+Install **Product Profile Example** module to see a use case in action.
+
+Profiles are also defined as search filter and group
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/product-attribute/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/product-attribute/issues/new?body=module:%20product_profile%0Aversion:%201.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+
+Credits
+=======
+
+Contributors
+------------
+
+* David BEAL <david.beal@akretion.com>
+* Sébastien BEAU <sebastien.beau@akretion.com>
+* Abdessamad HILALI <abdessamad.hilali@akretion.com>
+
+Iconography
```

Comment:
logo is in static folder, the right place for it (marketplace, and backoffice module view)